apiary.apib

```diff
@@ -1040,10 +1040,81 @@ This operations is available only for organization administrators.
 
 + Response 204
 
-## List Shared Buckets [/v2/storage/shared-buckets]
-Lists buckets which may be shared to the project.
+## Manage Shared Buckets [/v2/storage/shared-buckets]
+
+### Shared bucket detail [GET /v2/storage/shared-buckets/{project_id}/buckets/{bucket_id}/tables]
```

Comment:
Když tak na to koukám, tak tady vůbec nemusí být to tables ne? Nedává to smysl - detial bucketu by měl být na `/v2/storage/shared-buckets/{project_id}/buckets/{bucket_id}`, ne?